server/src/test/java/io/crate/execution/engine/aggregation/impl/CountAggregationTest.java

```diff
@@ -75,6 +121,215 @@ public void test_function_implements_doc_values_aggregator_for_string_based_type
         }
     }
 
+    private void test_count_on_object_with_not_null_immediate_child(DataType<?> childType, Class<?> expectedAggregatorClass) {
```

Comment:
Is this a test case? Misses the annotation and is private?

Reply:
also renamed, `helper_count_on_object_with_not_null_immediate_child`. I have a test that loops over that method